vcd/resource_vcd_nsxt_edgegateway_dhcp_forwarding_test.go

```diff
@@ -0,0 +1,192 @@
+//go:build network || nsxt || ALL || functional
+
+package vcd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccVcdNsxtEdgeDhcpForwarding(t *testing.T) {
+	preTestChecks(t)
+
+	// Requires VCD 10.3.1+
+	vcdClient := createTemporaryVCDConnection(true)
+	if vcdClient == nil {
+		t.Skip(acceptanceTestsSkipped)
```

Comment:
We should use a more informative skipping message

Reply:
Updated just like in the other case.